tests/ui/linkage-attr/unreferenced-used-static-issue-127052.rs

```diff
@@ -0,0 +1,9 @@
+// This is a non-regression test for issue #127052 where unreferenced `#[used]` statics couldn't be
+// removed by the MSVC linker, causing linking errors.
+
+//@ build-pass: needs linking
+//@ only-msvc
```

Comment:
Why shouldn't we run this test on all targets?

Reply:
The specific issue doesn't exist elsewhere so it's less useful there: the existing tests already cover `#[used]` features and targets more than this test. Of course, it wouldn't hurt. Feel free to change the directive to something you prefer.

Reply:
Well, clearly the existing tests do not cover `#[used]` in the binary, or my PR would not have landed.